src/main/java/core/basesyntax/model/OperationType.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.model;
+
+public enum OperationType {
+    BALANCE("b"),
+    SUPPLY("s"),
+    PURCHASE("p"),
+    RETURN("r");
+
+    private String code;
```

Comment:
According to the encapsulation principle, the 'code' field should be private to prevent direct access to it.